src/components/Common/SectionTitle/SectionTitle.tsx

```diff
@@ -9,13 +9,24 @@ export default function SectionTitle(props: SectionTitleSpec): JSX.Element {
 
 	return (
 		<div className='flex flex-col relative uppercase'>
-			{props.subtitle ?
-				<div className="z-10 absolute -top-4 -left-14 p-0.5 pl-2 pr-2 rounded-lg transform -rotate-12 bg-pink-dark-pink font-bold text-white-white">{props.subtitle}</div>
-				: ''}
+			{
+				props.subtitle ? (
+					<div className="z-10 absolute -top-4 -left-14 p-0.5 pl-2 pr-2 rounded-lg transform -rotate-12 bg-pink-dark-pink font-bold text-white-white">
+						{props.subtitle}
+					</div>
+				) : ''
+			}
```

Comment:
Should be:
```ts
{
    props.subtitle && (
        <div>...</div>
    )
}
```

Because right now, React is still trying to create a node (albeit empty).

Reply:
Right